config/action.d/iptables-allports.conf

```diff
@@ -21,6 +23,10 @@ actionstart = <iptables> -N f2b-<name>
               <iptables> -A f2b-<name> -j <returntype>
               <iptables> -I <chain> -p <protocol> -j f2b-<name>
 
+actionstart6 = <ip6tables> -N f2b-<name>6
+               <ip6tables> -A f2b-<name>6 -j <returntype>
+               <ip6tables> -I <chain> -p <protocol> -j f2b-<name>6
```

Comment:
I see no problem of having the same name (`f2b-<name>`) in both iptables and ip6tables -- those seems to be maintained independently... so what is the purpose of the 6 suffix?